benchmarks/source-sanity/update-post.js

```diff
@@ -18,7 +18,9 @@ console.log(`Getting article number ${randomDoc}`)
 client
   .fetch(`*[$randomDoc]`, { randomDoc })
   .then(({ _id, title }) => {
-    client.patch(_id).set({ title: title + "!" })
+    return client.patch(_id).set({ title: title + "!" })
+  })
```

Comment:
Tried running formatter but now realize that won't work.

Prettier/eslint autoformat should turn this function body into an expression. And the double quotes into backticks.

```suggestion
  .then(({ _id, title }) => client.patch(_id).set({ title: title + `!` }))
```

(Wouldn't use this suggestion, prettier may want to line break it, just saying this is kind of what I'm expecting here because we enforce expression arrows where possible.)